pkg/k8shandler/collection.go

```diff
@@ -72,7 +72,7 @@ func (clusterRequest *ClusterLoggingRequest) CreateOrUpdateCollection(proxyConfi
 		}
 
 		if err = clusterRequest.createOrUpdateFluentdPrometheusRule(); err != nil {
-			return
+			log.Error(err, "unable to create or update fluentd prometheus rule")
```

Comment:
This is a serious condition we should not simply drop it into the logs and carry on. It needs to be notified on status conditions and/or alerted. I will LGTM this since it's better than nothing, but we need to come back to this kind of error mis-handling and clean it up.

/lgtm